.env

```diff
@@ -56,6 +56,8 @@ REACT_APP_DEFAULT_LIVE_MONITORING=false
 ######################################################################################
 
 REACT_APP_CF_URL=http://localhost:8080/ChannelFinder
+REACT_APP_CF_PROPS_PATH=/resources/properties
```

Comment:
I think we can remove these two new env variables and make the assumption that channel finder endpoints will always be:
/resources/channels
/resources/properties
/resources/tags